src/lib/Backpack/devBackpack.cpp

```diff
@@ -258,14 +258,20 @@ static void AuxStateToMSPOut()
 #endif // USE_TX_BACKPACK
 }
 
-void crsfTelemToMSPOut(uint8_t *data)
+void sendCRSFTelemetryToBackpack(uint8_t *data)
 {
-    if (config.GetBackpackTlmEnabled() == 0)
+    if (config.GetBackpackTlmMode() == 0)
```

Comment:
Isn't there an enum for this? I know you hate magic numbers 😆

Reply:
There isn't currently, but I can add one.